### ISSUE TITLE
the evaluation of dair-v2x-i

### ISSUE THREAD

**Issue by wanghangege** [ADLab-AutoDrive/BEVHeight#8]
![image](https://user-images.githubusercontent.com/51116270/232942752-026e1dbb-b1b8-4c6d-a0d1-bcb4857e4114.png)
hello, bro yanglei! I want to know the implementation method of pointpillar in the table below, for example, are you testing based on openpcdet? The indicators you count are: bbox, bev, 3d or aos? I guess it is 3d_R40, consistent with your Rope data? Please help me to confirm my question, thank you.

**Comment by wanghangege** [comment]
Another question, in other methods of detection, what is the point cloud detection range you set (lidar coors, x and y)? This is related to the fairness of lidar and image comparison and evaluation in 3D solutions. I assume that it is consistent with the BEVHeight in the text, which is [104, 104]
![image](https://user-images.githubusercontent.com/51116270/232944480-b75f186d-55b8-4475-b119-8b6c87ee87eb.png)

**Comment by yanglei18** [comment]
@wanghangege 
we adopted the metric AP3D_R40 for all experiments. Please refer to "DAIR-V2X: A Large-Scale Dataset for Vehicle-Infrastructure Cooperative 3D Object Detection" for details about the implementation method of pointpillar et. al.

![image](https://user-images.githubusercontent.com/82928820/232989622-5a8ef744-583c-4f97-a0f5-5a71ac3b0cd1.png)

**Comment by yanglei18** [comment]
@wanghangege 
We set up the same point cloud detection range for BEVFormer, BEVDepth, and BEVHeight detectors.
```
'x_bound': [0, 102.4, 0.8],
'y_bound': [-51.2, 51.2, 0.8],
'z_bound': [-5, 3, 8],
```

**Comment by wanghangege** [comment]
tanks for your reply, I have known your baseline inplementedin MMdetection3d, but have you use some pretrained models for lidar detection baselines ? I use the openpcdet framwork to train pointpillar (with pretrained models and some easy data augmentation，lidar range with [0, -51.2, -3, 102.4, 51.2, 2]), and I have achieved AP3D_R40 with nearly 80,  not 63.07 in your papers. As we all know, dair-v2x-i include many types of "Car" and "Cyclist", such as this picture:
![image](https://user-images.githubusercontent.com/51116270/233760558-2ad4fc7a-57e5-454b-a59b-2f32dc7b4f4a.png)
So ,I want to know  if you incorporated the relevant classes into "Car", "Cyclist", and "pedestrian", or just train your model without pretrained model and data augmentation? Looking forward to your reply.

**Comment by wanghangege** [comment]
the results of pointpillar at dair-v2x-i like this:
<img width="245" alt="1682213284865" src="https://user-images.githubusercontent.com/51116270/233814427-b7cb4082-b1d2-431c-a61c-48cf156084c4.png">

**Comment by yanglei18** [comment]
@wanghangege 
**1. For the metrics of Poinpillars, ImvoxelNet, SECOND, and MVXNet detectors, we follow the results of the official dataset paper "DAIR-V2X: A Large-Scale Dataset for Vehicle-Infrastructure Cooperative 3D Object Detection". Please read this paper carefully for more details. 
![image](https://user-images.githubusercontent.com/82928820/233844789-e61034fb-59fc-4d23-b19f-270d93f65e38.png)

**2. The vehicle contains Car, Truck, Van, and Bus. The Cyclist includes Cyclist, Tricyclist, Motorcyclist, and Barrowlist. The Pedestrian is only for the Pedestrian

**Comment by wanghangege** [comment]
I know I've asked too many basic questions, which makes me feel embarrassed, but I really want to ask the last one.
![image](https://user-images.githubusercontent.com/51116270/233849574-c2bbb8a6-3e65-46e6-9248-feae41507f01.png)
 Have you found this classification standard in their paper (in fact, I haven't seen it), and it's also difficult for me to find it in their source code, but the Car type did briefly mention (https://github.com/AIR-THU/DAIR-V2X/blob/main/data/README_zh-CN.md).
![image](https://user-images.githubusercontent.com/51116270/233849744-519af72b-01c4-44d5-837c-be0e86b3ae8a.png).
 If you could point out the classification basis, I would greatly appreciate it and sincerely wish your research more and more smoothly!
 In addition, their range based on point clouds [0, -39.68, -3, 69.12, 39.68, 1] is not as far away as in your experiment_ bound': [0, 102.4, 0.8],'y_ bound': [-51.2, 51.2, 0.8],'z_ bound': [-5, 3, 8]。 This indicates that your work is much better than expected![https://github.com/AIR-THU/DAIR-V2X/blob/main/configs/sv3d-inf/imvoxelnet/trainval_config.py]

**Comment by yanglei18** [comment]
**1. The detailed benchmark is available on the official website (https://thudair.baai.ac.cn/index) some time ago. But the information cannot be found now. 
**2. Tsinghua AIR - Baidu Apollo Challenge on Vehicle-Infrastructure Cooperative Perception follows the same evaluation basis, which can be referred as "https://aistudio.baidu.com/aistudio/competition/detail/522/0/task-definition"
![image](https://user-images.githubusercontent.com/82928820/233910966-6d165d89-6702-4e46-b483-1e0b1e856aa4.png)

**Comment by Ignite616** [comment]
> tanks for your reply, I have known your baseline inplementedin MMdetection3d, but have you use some pretrained models for lidar detection baselines ? I use the openpcdet framwork to train pointpillar (with pretrained models and some easy data augmentation，lidar range with [0, -51.2, -3, 102.4, 51.2, 2]), and I have achieved AP3D_R40 with nearly 80, not 63.07 in your papers. As we all know, dair-v2x-i include many types of "Car" and "Cyclist", such as this picture: ![image](https://user-images.githubusercontent.com/51116270/233760558-2ad4fc7a-57e5-454b-a59b-2f32dc7b4f4a.png) So ,I want to know if you incorporated the relevant classes into "Car", "Cyclist", and "pedestrian", or just train your model without pretrained model and data augmentation? Looking forward to your reply.

Hello, where is the baseline of BEVHeight in MMdetection3d, I hope to use it for experiments. thank you!

**Comment by microchinajie** [comment]
Based on my own experiments, the difference of point cloud range leads to dramatic difference of detection performance. The point cloud range of PointPillars implemented by DAIR-V2X is [0, -39.68, -3, 69.12, 39.68, 1], while this paper is [0, -70.4, -5, 140.8, 70.4, 3]. Therefore, I think this is unfair comparison of different methods in validation split in Table 1.
